eip-5564.md

```diff
@@ -178,12 +198,13 @@ An example function to check whether a stealth address belongs to oneself:
 - The  `checkStealthAddress` function performs the following computations:
   - Shared secret $s$ is computed by multiplying the viewing private key with the ephemeral public key of the announcement $s = p_{view}$ * $P_{ephemeral}$.
   - The secret is hashed $s_{h} = h(s)$.
+  - The view tag $v$ is extracted by taking the most significant byte $s_{h}[0]$ and can be compared to the given view tag.
   - Multiply the hashed shared secret with the generator point $S_h = s_h \cdot G$.
   - The stealth public key is computed as $P_{stealth} = P_{spend} + S_h$.
   - The derived stealth address $a_{stealth}$ is computed as $\textrm{pubkeyToAddress}(P_{stealth})$.
   - Return `true` if the stealth address of the announcement matches the derived stealth address, else return `false`.
 
-An example function to derive the private key of a stealth address:
+Private key derivation - Generate the stealth address private key from the hashed shared secret and the spending private key.
 
 - User has access to the viewing private key, spending private key $p_{spend}$.
 
```

Comment:
Github won't let me comment on this line, but a few lines below it says:

```
- The stealth private key is computed as $P_{stealth} = p_{spend} + s_h$.
```

but it should say:

```
- The stealth private key is computed as $p_{stealth} = p_{spend} + s_h$.
```

i.e. change the capital `P` to a lowercase `p` since it's a private key